runtime/services/catalog/artifacts/yaml/objects.go

```diff
@@ -338,6 +339,16 @@ func fromMetricsViewArtifact(metrics *MetricsView, path string) (*drivers.Catalo
 		}
 	}
 
+	// backwards compatibility where name was used as property
+	for i, dimension := range apiMetrics.Dimensions {
+		if dimension.Name == "" {
+			dimension.Name = fmt.Sprintf("dimension_%d", i)
+		}
+		if dimension.Property == "" {
+			dimension.Property = dimension.Name
+		}
```

Comment:
If `name == ""`, it will try to call `dimension_%d` in the underlying model during validation. That might be weird?